### ISSUE TITLE
Jigzi Player: "History of the Jews of England" Game not converted properly

### ISSUE THREAD

**Issue by chayaml** [ji-devs/ji-cloud#2453]
The Jigzi version of this game didn't convert well. The voiceover questions cannot be heard, so the game cannot be played. Also, the text on the bottom right of the screen ("King Edward") is oversized and hides the rest of the content...
What may be the problem?
**Jitap:**
![Screen Shot 2022-02-23 at 5 40 37 PM](https://user-images.githubusercontent.com/78186896/155354376-d4db1afe-3769-4f97-b837-d300a11764f3.png)
https://jitap.net/activities/gzb/play/

**Jigzi:**
![Screen Shot 2022-02-23 at 5 41 43 PM](https://user-images.githubusercontent.com/78186896/155354609-3b1b6409-019b-4d6c-9715-00545fef1ad7.png)
https://jigzi.org/admin/curation/750fd720-69d1-11ec-a181-cb0699d43ddd

**Comment by corinnewo** [comment]
@dakom Please take a look at this - it seems to be happening with ask a question with multiple questions on one screen. This one has background music, which might be a factor?

**Comment by chayaml** [comment]
closing in favor of #2471 and #2454
